hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/AbstractHoodieClient.java

```diff
@@ -132,9 +132,14 @@ protected void initWrapperFSMetrics() {
   }
 
   protected HoodieTableMetaClient createMetaClient(boolean loadActiveTimelineOnLoad) {
-    return HoodieTableMetaClient.builder().setConf(hadoopConf).setBasePath(config.getBasePath())
-        .setLoadActiveTimelineOnLoad(loadActiveTimelineOnLoad).setConsistencyGuardConfig(config.getConsistencyGuardConfig())
-        .setLayoutVersion(Option.of(new TimelineLayoutVersion(config.getTimelineLayoutVersion()))).build();
+    return HoodieTableMetaClient.builder()
```

Comment:
could we please avoid non-essential style fixes in the PR. Makes it harder to review.

Reply:
Changed :)